### DISCUSSION TITLE
logfile/out put in docker

### DISCUSSION THREAD

**Discussion by simonmcnair** [beetbox/beets#4666] [General]
Hi,
I run Beets in a docker container provided by linuxserver.io .  I'm interested if I can see progress/activity in docker logs, or more detail in the log file as, at the moment, I can see it working, but don't know how far through the library it is ?

Love it btw, thanks all of those responsible for it.

**Comment by wisp3rwind** [comment]
> [...] I run Beets in a docker container provided by linuxserver.io [...]

You should ask at linuxserver.io, then. They'll certainly be more knowledgeable about docker than anyone here.

**Reply by simonmcnair** [reply]
Thanks for responding.  They can only output the result of the container.  I guess my question is more along the lines of if there is a plugin for Beets that can provide a progress bar or ETA for completion.

**Comment by sampsyo** [comment]
Maybe you could clarify exactly what configuration and options you're using for the import? By default, beets prints out plenty of stuff during the import, so it's not clear what more you are looking for.

**Reply by simonmcnair** [reply]
My apologies for the lack of clarity.  I guess, if I were to describe an enhancement, it would be to get a total number of files to import at start, then starting at 1, increment for each file in the log so that I can see how far through the import it is.  Optionally a percentage would be nice, and an eta based on the time the last last process took.
I'm sure there are a multitude of reasons why this is impractical or difficult, but I would just like an indication of how fast it is going, and when I can expect the import to complete.

Thanks for a great piece of software.

**Reply by sampsyo** [reply]
Ah, right, an actual numerical counter like that is not something that's built into beets. You can imagine running `beet ls | wc -l`, however, to list the number of files in your library (concurrently, while an import is running); this would give you an idea of what progress has been completed.
